docker-compose.yml

```diff
@@ -2,18 +2,16 @@ version: "3.7"
 
 services:
   ssh-log-to-influx:
-    image: acouvreur/ssh-log-to-influx
+    image: rolfrander/geohashlog
```

Comment:
Not so sure about that...

Reply:
:-)

Reply:
Ok, I don't know how to handle this. I might continue adding features to my branch independently, and I would like to use my own name for the image.

Reply:
I don't either, I'm afraid you'll have to handle multiple branches and do not create pull requests from your master branch anymore ?

Reply:
I need to learn more advanced git...

Reply:
Can you revert the changes for the PR ?